backend/api/tests/schema/test_submission.py

```diff
@@ -47,3 +50,28 @@ def test_returns_correct_submission(graphql_client, user, submission_factory):
 
     assert not resp.get("errors")
     assert resp["data"]["submission"]["id"] == str(submission.id)
+
+
+@pytest.mark.django_db
+def test_submission_tag_get_or_create(graphql_client, user, submission_tag_factory):
+    graphql_client.force_login(user)
+
+    query = """
+        mutation($name: String!) {
+            sendTag(input: {
+                name: $name
+            }) {
+                __typename
+                ... on SubmissionTag{
+                    name
+                }
+                ... on SendTagErrors {
+                    nonFieldErrors
+                }
+            }
+        }
```

Comment:
why do we need a send tag mutation? can't we do that when we send/edit a submission?

Reply:
or is this to create new tags? I think we should allow only a few tags decided by us :)

Or if we want to allow any kind of tags I guess at that point we can use an array field instead of having a model for the tags :)

Reply:
No, I don't know what I was thinking... Maybe when in the frontend you hit the comma and appears the tag-button I think to do a mutation to save that tag initially... 
I removed it